config/initializers/time_with_zone.rb

```diff
@@ -0,0 +1,14 @@
+# Extends ActiveSupport::TimeWithZone class with convenience methods
+module ActiveSupport
+  class TimeWithZone
+    def display_date
+      tz = Config.time_zone || Rails.application.time_zone
```

Comment:
I think this might need to be `Rails.application.config.time_zone`? a quick snippet from a shell in prod:

```
irb(main):003:0> Rails.application.time_zone
(irb):3:in `<main>': undefined method `time_zone' for #<DARIA::Application> (NoMethodError)
irb(main):004:0> Rails.application.config.time_zone
=> "Eastern Time (US & Canada)"
```

maybe this works differently in context though? 


The other thing worth noting is that `Config.time_zone` will be scoped to a given fund based on the subdomain. but since I think that gets eval'd at showtime, should be cool.

Reply:
Great catch, total typo on my part. Thanks!